lib/loading_screen.dart

```diff
@@ -0,0 +1,67 @@
+import 'package:distal_radius/image_upload_screen.dart';
+import 'package:flutter/material.dart';
+
+import 'package:camera/camera.dart';
+import "camera_screen.dart";
+import 'screen_button.dart';
+import "menu_screen.dart";
+
+class LoadingScreen extends StatefulWidget {
+  const LoadingScreen({Key? key}) : super(key: key);
+
+  //String to be used in navigator
+  static const String id = "loading_screen";
+
+  @override
+  State<LoadingScreen> createState() => _LoadingScreenState();
+}
+
+class _LoadingScreenState extends State<LoadingScreen> {
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      body: SafeArea(
+        child: Container(
+          margin: const EdgeInsets.fromLTRB(0, 35, 0, 0),
+          child: Column(
+            crossAxisAlignment: CrossAxisAlignment.stretch,
+            children: <Widget>[
+              Column(
+                mainAxisAlignment: MainAxisAlignment.center,
+                children: <Widget>[
+                //back button
+                  Stack(alignment: Alignment.center, children: const [
+                    Positioned(left: 10, child: BackButton()),
+                    Align(
+                        child: Text(
+                          "Loading Screen",
```

Comment:
Small detail: don't need to add "Screen" to the title, so that it's more concise for the user.

```suggestion
                          "Loading",
```